go-client/pegasus/table_connector.go

```diff
@@ -715,8 +719,29 @@ func (p *pegasusTableConnector) Incr(ctx context.Context, hashKey []byte, sortKe
 }
 
 func (p *pegasusTableConnector) runPartitionOp(ctx context.Context, hashKey []byte, req op.Request, optype OpType) (interface{}, error) {
+	start := time.Now()
+	var errResult error
+	if p.enableMetrics {
+		defer func() {
+			elapsed := time.Since(start).Nanoseconds()
+			pm := metrics.GetPrometheusMetrics()
+			status := "success"
+			if errResult != nil {
+				if errors.Is(ctx.Err(), context.DeadlineExceeded) {
+					status = "timeout"
+				} else {
+					status = "fail"
+				}
+			}
+			meta := strings.Join(p.meta.GetMetaIPAddrs(), ",")
+			pm.MarkMeter(fmt.Sprintf("pegasus_client_%s_%s_%s_total", p.tableName, optype.String(), status), 1, map[string]string{"meta": meta})
```

Comment:
How about add more labels rather than name components? the labels in Prometheus is more flexible, we can filter or aggregation metrics by labels.